### ISSUE TITLE
WinForms Tooltips should become visible when hover over

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#2726]
* .NET Core Version: 
5.0

* Have you experienced this same bug with .NET Framework?: 
Yes

**Problem description:**

WinForms ToolTip should meet WCAG 2.1 requirements and satisfy the rule 1.4.13 "Content on Hover or Focus". For more details please see [WCAG 2.1 Criterion 1.4.13](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html)

**Expected behavior:**
ToolTip should become visible when hover over.

**Minimal repro:**
1. Create WinForms app with some control and ToolTip.
2. Add the ToolTip text for the control.
3. Run app, mouse over the control. ToolTip appears after a second.
4. Mouse over the ToolTip (mouse should be out of the control). Observe: ToolTip disappears.

**Comment by jlukosky** [comment]
To be compliant with 1.4.13 the following needs to be supported:
•	Dismissible: A mechanism is available to dismiss the additional content without moving pointer hover or keyboard focus, unless the additional content communicates an input error or does not obscure or replace other content.
•	Hoverable: If pointer hover can trigger the additional content, then the pointer can be moved over the additional content without the additional content disappearing.
•	Persistent: The additional content remains visible until the hover or focus trigger is removed, the user dismisses it, or its information is no longer valid.

The expected behavior and minimal repro listed should handle the hoverable aspect but not the dismissable or persistent.

Are those required aspects being addressed as well in this or another issue or have are they handled and I'm not finding the documentation?

**Comment by merriemcgaw** [comment]
@jlukosky we have the new tooltip requirements are are working towards meeting these criteria. Under the hood, WinForms is dependent on the Windows Common Controls for much of our functionality. We're engaging directly with that team to provide us a base ToolTip control that supports the requirements so that we can then expose that to the .NET WinForms customer. This particular issue is one which we can control directly, and we're working on ensuring all controls that support ToolTips support them from keyboard focus while the Common Controls team is working on the underlying layer.

Thanks!

**Comment by jlukosky** [comment]
@merriemcgaw - That's exactly what I needed to know. Thanks for the rapid response.

**Comment by merriemcgaw** [comment]
@jlukosky, @RussKie deserves all the credit as he called this question to my attention 😄

**Comment by Tanya-Solyanik** [comment]
duplicate of https://github.com/dotnet/winforms/pull/5424
